v5_order_service.go

```diff
@@ -231,39 +233,41 @@ type V5GetOrdersResult struct {
-	UpdatedTime        string      `json:"updatedTime"`
-	Side               Side        `json:"side"`
-	TriggerPrice       string      `json:"triggerPrice"`
-	CumExecFee         string      `json:"cumExecFee"`
-	LeavesQty          string      `json:"leavesQty"`
-	SlTriggerBy        string      `json:"slTriggerBy"`
-	CloseOnTrigger     bool        `json:"closeOnTrigger"`
-	CumExecQty         string      `json:"cumExecQty"`
-	ReduceOnly         bool        `json:"reduceOnly"`
-	Qty                string      `json:"qty"`
-	StopLoss           string      `json:"stopLoss"`
-	TriggerBy          TriggerBy   `json:"triggerBy"`
+	Symbol                SymbolV5              `json:"symbol"`
+	OrderType             OrderType             `json:"orderType"`
+	OrderLinkID           string                `json:"orderLinkId"`
+	OrderID               string                `json:"orderId"`
+	CancelType            string                `json:"cancelType"`
+	AvgPrice              string                `json:"avgPrice"`
+	StopOrderType         string                `json:"stopOrderType"`
+	LastPriceOnCreated    string                `json:"lastPriceOnCreated"`
+	OrderStatus           OrderStatus           `json:"orderStatus"`
+	TakeProfit            string                `json:"takeProfit"`
+	CumExecValue          string                `json:"cumExecValue"`
+	TriggerDirection      int                   `json:"triggerDirection"`
+	IsLeverage            string                `json:"isLeverage"`
+	RejectReason          string                `json:"rejectReason"`
+	Price                 string                `json:"price"`
+	OrderIv               string                `json:"orderIv"`
+	CreatedTime           string                `json:"createdTime"`
+	TpTriggerBy           string                `json:"tpTriggerBy"`
+	PositionIdx           int                   `json:"positionIdx"`
+	TimeInForce           TimeInForce           `json:"timeInForce"`
+	LeavesValue           string                `json:"leavesValue"`
+	UpdatedTime           string                `json:"updatedTime"`
+	Side                  Side                  `json:"side"`
+	TriggerPrice          string                `json:"triggerPrice"`
+	CumExecFee            string                `json:"cumExecFee"`
+	LeavesQty             string                `json:"leavesQty"`
+	SlTriggerBy           string                `json:"slTriggerBy"`
+	CloseOnTrigger        bool                  `json:"closeOnTrigger"`
+	CumExecQty            string                `json:"cumExecQty"`
+	ReduceOnly            bool                  `json:"reduceOnly"`
+	Qty                   string                `json:"qty"`
+	StopLoss              string                `json:"stopLoss"`
+	TriggerBy             TriggerBy             `json:"triggerBy"`
+	SlippageToleranceType SlippageToleranceType `json:"slippageToleranceType"`
+	SlippageTolerance     string                `json:"slippageTolerance"`
```

Comment:
`V5GetOrder` is referenced by both `GetOpenOrders` and `GetHistoryOrders`, but only `GetHistoryOrders` includes `SlippageToleranceType` and `SlippageTolerance` in the response.

It might be better to separate the types.

docs:
- https://bybit-exchange.github.io/docs/v5/order/open-order
- https://bybit-exchange.github.io/docs/v5/order/order-list